build/props.js

```diff
@@ -18,7 +18,14 @@ import {toJSON} from './to-json.js'
 import {toTokens} from './to-tokens.js'
 import {toFigmaTokens} from './to-figmatokens.js'
 
-const [,,prefix,useWhere] = process.argv
+let prefix = process.argv[2];
+const useWhere = process.argv[3]
+
+// On windows devices the prefixs is "''" if not set
+if(prefix === "''") {
+  prefix = undefined;
+}
+
```

Comment:
```suggestion
const [,,prefix='',useWhere] = process.argv
```

then in [`to-stylesheet.js`](https://github.com/argyleink/open-props/blob/main/build/to-stylesheet.js#L17) update that prefix if statement to check for empty strings too? normalizes empty string as the default.

Reply:
Did some changes, didn't quite get what you were trying to tell me., but I realized, that there is already a check for this in [to-stylesheet.js](https://github.com/argyleink/open-props/blob/main/build/to-stylesheet.js#L17) as you said. Thats why I just added the check in there, does the same, it is much cleaner.